Formula/d/devtunnel.rb

```diff
@@ -0,0 +1,25 @@
+class Devtunnel < Formula
+  desc "Microsoft Devtunnel allow developers to securely share local web services"
+  homepage "https://aka.ms/devtunnels/docs"
+  url "https://tunnelsassetsprod.blob.core.windows.net/cli/1.0.922+fa34179138/osx-x64-devtunnel-zip"
```

Comment:
https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae

Reply:
@SMillerDev , so instead of formula should it be a cask ?

Reply:
I don't know, can it not be build from source?